IdentityCore/src/configuration/MSIDConfiguration.m

```diff
@@ -33,27 +34,15 @@ - (instancetype)copyWithZone:(NSZone*)zone
     configuration.redirectUri = [_redirectUri copyWithZone:zone];
     configuration.target = [_target copyWithZone:zone];
     configuration.clientId = [_clientId copyWithZone:zone];
-    configuration.correlationId = [_correlationId copyWithZone:zone];
-    configuration.loginHint = [_loginHint copyWithZone:zone];
-    configuration.sliceParameters = [_sliceParameters copyWithZone:zone];
```

Comment:
I think we need slice parameters for all requests, both interactive and silent

Reply:
I think it is to be determined whether it will use MSIDConfiguration class or not.